tests/unit/testTypesService.unitTest.ts

```diff
@@ -91,6 +91,23 @@ describe("getTestTypesList", () => {
               });
           });
         });
+
+        context("when vehicleSize, vehicleAxles and vehicleConfiguration filters are not present", () => {
```

Comment:
Looks good, but not sure how many of the "does have filter" permutations are tested for. Might be worth just double checking that each of the filters is being fully tested - sonar can probably tell you quickly and easily.
The "testCodes.length > 1" test probably covers you pretty well